afids_utils/tests/test_io.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import annotations
+
+import csv
+import tempfile
+from os import PathLike, remove
+from pathlib import Path
+
+import numpy as np
+import pytest
+from hypothesis import HealthCheck, assume, given, settings
+from hypothesis import strategies as st
+from numpy.typing import NDArray
+
+from afids_utils.exceptions import InvalidFiducialNumberError
+from afids_utils.io import AFIDS_FIELDNAMES, afids_to_fcsv, get_afid, load_nii
+
+
+@pytest.fixture
+def nii_file() -> PathLike[str]:
+    return (
+        Path(__file__).parent
+        / "data"
+        / "tpl-MNI152NLin2009cAsym_res-01_desc-brain_T1w.nii.gz"
+    )
+
+
+class TestNiftiIO:
+    def test_load_valid_path(self, nii_file: PathLike[str]):
```

Comment:
This'll probably change anyway, but this should check that all the values in `nii_data` are between 0 and 1.

Reply:
Ahh yeah, didn't think about that.

Reply:
Ended up removing this all together since `load_nii` has been rremoved.